docs/guidelines/content/set-plagiarism.md

```diff
@@ -12,7 +12,7 @@ Achievement and mastery badges need to be unique from other sets. Slightly modif
 
 ### Titles
 
-Achievement titles should be unique from other sets. This may not always be possible when working on games that have ports for multiple systems. Generic titles and titles based on the in-game contents are both allowed. For example, Green Hills Zone is the first level of Sonic the Hedgehog on the Mega Drive, Master System, and Game Gear ports. Additionally, the level shows up in later entries of the Sonic series. As such, the achievement title “Green Hills Zone” would be acceptable in all these sets.
+Achievement titles should be unique from other sets. This may not always be possible when working on games that have ports for multiple systems. Generic titles and titles based on the in-game contents are both allowed. For example, Green Hill Zone is the first level of Sonic the Hedgehog on the Mega Drive/Genesis, Master System and Game Gear versions. Additionally, the level shows up in later entries of the Sonic series. As such, the achievement title “Green Hill Zone” would be acceptable in all these sets.
```

Comment:
Any reason to remove the Oxford comma here?

Reply:
For the sake of consistency, ~30 pages of the docs use the Oxford comma.

![Screenshot 2024-08-22 at 11 54 25 AM](https://github.com/user-attachments/assets/95654306-9d2b-42e6-82ff-203022cabece)

Reply:
Oh, I'm sorry, I wasn't aware of that. I'll add it back real quick.

Reply:
https://github.com/SuperGamer18/RetroAchievements-docs/blob/updates/docs/guidelines/content/set-plagiarism.md gotcha

Reply:
Sorry, it looks like the changes didn't land in this branch. You'll need to make the change in this branch for the PR to automatically capture the update.

Reply:
https://github.com/SuperGamer18/RetroAchievements-docs/blob/patch-1/docs/guidelines/content/set-plagiarism.md

Reply:
Yeah, I'll try to use the updates branch instead of this branch that GitHub automatically generated for me next time, if possible (I would like to do more edits here but I don't want to overblow this repo with PRs almost exclusively from me, and I also wanted to do something since that other PR of mine)

Reply:
Sorry, I'm not sure I'm articulating this well.

The comma issue is still happening. When you made [this edit](https://github.com/RetroAchievements/docs/pull/91#discussion_r1727420790), it was done in your _main_ branch, not your _patch-1_ branch. This PR is coming from _patch-1_, and until the issue is fixed in _patch-1_, it will not be fixed in the PR.

This is the file in _patch-1_:
https://github.com/SuperGamer18/RetroAchievements-docs/blob/patch-1/docs/guidelines/content/set-plagiarism.md

You need to fix the comma issue in this file. When it's fixed there, the PR will automatically get the change.

Reply:
I was already editing that branch as you said that, sorry. The branch I mainly use for updates is the *updates* branch as you're trying to tell me, but since I haven't done a PR in a while and forgot to change the branch we have to deal with this issue. And indeed, the PR got the changes.